test-cases/longevity/longevity-per-table-hints-1h.yaml

```diff
@@ -0,0 +1,42 @@
+# This test validates the per-table hints functionality with the expected_data_size_in_gb and
+# min_per_shard_tablet_count option.
+# The test creates a table with tablets configuration and runs a mixed workload with GrowShrinkNemesis
+# to test hint behavior during cluster scaling operations.
+#
+# Test Evaluation:
+# - Results must be evaluated manually from Grafana keyspace charts
```

Comment:
need instructions what to evaluate
if instructions are provided - can it be automated?